### ISSUE TITLE
[Breaking]: Mod will not load

### ISSUE THREAD

**Issue by Brinjoh** [Annosz/UIInfoSuite2#429]
### Mod Version

v2.3.0

### Multiplayer

Multiplayer (Non-Host)

### SMAPI Log

https://smapi.io/log/b95d079e930141cd9ad683a1dc552b5d

### Issue
SMAPI continues to tell me UI Info Suite 2 is no longer compatible and to download the updated version even though I have downloaded the updated version and installed it. Reinstalling stardew valley, smapi, and the mod doesn't fix the issue. It occurs on loading the game with SMAPI regardless of single player or multiplayer, and in multiplayer my host partner is having no issues with the mod.

### Steps to Reproduce
1. Install Stardew Valley
2. Install SMAPI
3. Install UI Info Suite
4. Load Stardew Valley with SMAPI

**Comment by drewhoener** [comment]
This is interesting to, I'm pretty sure I'm using the same versions of everything as you, did you just install SMAPI today? 
Have you tried just running it on its own with any other mods? Maybe there's some compatibility that's breaking it

**Comment by Brinjoh** [comment]
No, from what was above I think SMAPI had been installed yesterday.

Just now I downloaded and reinstalled SMAPI and UI Info Suite fresh, started up Stardew Valley with SMAPI and only this mod and the problem persisted.

But this did lead me to asking my host partner some questions and checking Steam's settings in greater detail. This is really very odd, but in the past I had Stardew Valley installed on drive F:/ but I installed the new 1.6 update to C:/ instead. So I had some copy of Stardew Valley with a variety of mods on both F:/ and C:/.

When I was having this problem, my Steam settings were:
Target: "c:\program files (x86)\steam\steamapps\common\Stardew Valley\StardewModdingAPI.exe" %command%
Start in: 'F:\SteamLibrary\steamapps\common\Stardew Valley'

What fixed it was changing "Start in" to point to C:\
Target: "c:\program files (x86)\steam\steamapps\common\Stardew Valley\StardewModdingAPI.exe" %command%
Start in: C:\Program Files (x86)\Steam\steamapps\common\Stardew Valley

This is pretty confusing.

I started up a new single player game to test and it's all working great, here's the SMAPI log
https://smapi.io/log/e910762ca87b49339f3e6c5c4a25b992
